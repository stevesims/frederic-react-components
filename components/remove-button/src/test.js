import React from 'react';
import { mount, shallow } from 'enzyme';
import { createMatchers } from 'jest-emotion';
import * as emotion from 'emotion';

import RemoveButton from '.';

expect.extend(createMatchers(emotion));

describe('RemoveButton', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<RemoveButton />);
    expect(wrapper.exists()).toBe(true);
  });

  it('matches snapshot', () => {
    const wrapper = mount(<RemoveButton />);
    expect(wrapper).toMatchSnapshot();
  });
});
