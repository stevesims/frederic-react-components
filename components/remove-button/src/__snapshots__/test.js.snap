// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RemoveButton matches snapshot 1`] = `
.emotion-0 {
  background-color: #0C60A2;
  border: 0;
  color: #fff;
  font-family: "nta",Arial,sans-serif;
  font-size: 11px;
  font-weight: 400;
  height: 20px;
  line-height: normal;
  margin: 0 12px;
  padding: 0;
  width: 20px;
}

.emotion-0:first-child {
  margin: 0;
}

<RemoveButton>
  <RemoveButtonInner>
    <button
      className="emotion-0 emotion-1"
    >
      X
    </button>
  </RemoveButtonInner>
</RemoveButton>
`;
